deps/wazuh_testing/wazuh_testing/tools/manager_keys.py

```diff
@@ -0,0 +1,49 @@
+sslmanager_key = """-----BEGIN PRIVATE KEY-----
```

Comment:
This file can be moved to the `data` directory and load them in the `tools` module.